lib/averages.js

```diff
@@ -0,0 +1,37 @@
+var pry = require('pryjs');
+
+var averages = module.exports = {
+  total: function total(collection) {
+    return collection.reduce(function(base, num) {
+      return base + num
+    })
+  },
+  mean: function mean(collection) {
+    return averages.total(collection)/(collection.length)
+  },
+  median: function median(collection) {
+    if (collection.length % 2 == 0) {
+      var i = (collection.length/2)
+      return (collection[i-1] + collection[i])/2
+    } else {
+      var i = Math.floor(collection.length/2)
+      return collection[i]
+    }
+  },
+  mode: function mode(collection) {
+    var maxKey = undefined
```

Comment:
Typically we don't want to set variables to undefined. instead you can just write `var maxKey`